js/web/lib/wasm/wasm-core-impl.ts

```diff
@@ -328,7 +328,7 @@ export const run =
                 errorCode = wasm._OrtGetTensorData(
                     tensor, tensorDataOffset, tensorDataOffset + 4, tensorDataOffset + 8, tensorDataOffset + 12);
                 if (errorCode !== 0) {
-                  throw new Error(`Can't get a tensor data. error code = ${errorCode}`);
+                  throw new Error(`Can't access tensor data - recieved unexpected data type. error code = ${errorCode}`);
```

Comment:
the failure here does not necessarily mean unexpected data type. actually all possible errors from OrtGetTensorData() can get to here